python_modules/dagster/dagster/_core/definitions/assets.py

```diff
@@ -315,6 +318,12 @@ def __init__(
         )
 
         self._is_subset = check.bool_param(is_subset, "is_subset")
+        self._owners_by_key = check.opt_mapping_param(
+            owners_by_key, "owners_by_key", key_type=AssetKey, value_type=list
+        )
+
+        if self._owners_by_key and any(self._owners_by_key.values()):
```

Comment:
nit: consolidate check
```suggestion
        if any(self._owners_by_key.values()):
```